github/retag-and-push.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Pushes a libero container image to Docker Hub with tags"
+    echo 
+    echo "Usage: $0 project input_tag"
+    echo "Example: $0 dummy-api 6cd9d7ebad4f16ef7273a7a831d79d5d5caf4164"
+    echo "Relies on the following environment variables:"
+    echo "- GITHUB_REF, GITHUB_SHA (GH Action default)"
+    echo "- DOCKER_USERNAME, DOCKER_PASSWORD"
+    exit 1
+fi
+
+project="$1"
+image_tag="$2"
+
+
+# login
+DOCKER_REGISTRY="${DOCKER_REGISTRY:-}"
+echo "${DOCKER_PASSWORD}" | docker login --username "${DOCKER_USERNAME}" --password-stdin "${DOCKER_REGISTRY}"
+
+# `libero/` currently not available, replace with `liberoadmin` for now
+name="${DOCKER_REGISTRY}liberoadmin/${project}"
+input_image="libero/${project}:${image_tag}"
+if ! (docker image inspect "$input_image" > /dev/null ) ; then
+    echo "Replacing libero/ with liberoadmin/"
+    input_image="liberoadmin/${project}:${image_tag}"
+fi
+
+if [[ "$GITHUB_REF" == "refs/heads"* ]]; then
+    # push `branch-sha` tagged image
+    branch="${GITHUB_REF/refs\/heads\//}"
+    docker tag "$input_image" "${name}:${branch}-${GITHUB_SHA}"
```

Comment:
Suggestion - using the whole SHA might become cumbersome, we could just use a shortened version?

Reply:
This would be a change to the [ADR](https://github.com/libero/community/blob/master/doc/adr/0009-container-image-tags.md). IMHO manipulating SHAs is done by copy/paste and the full-sha is more inline with how things were previously tagged.

I'd suggest raising this at the next sync?